coccinelle/safe_close.cocci

```diff
@@ -3,17 +3,17 @@
 expression fd;
 @@
 - close(fd);
-- fd = -1;
+- fd = -EBADF;
 + fd = safe_close(fd);
 @@
 expression fd;
 @@
 - close_nointr(fd);
-- fd = -1;
+- fd = -EBADF;
```

Comment:
i guess one could write an expression here that allows either -1 or -EBADF in the match expression. but i too would be too lazy for that ;-)